src/components/AppBar/NavBar.tsx

```diff
@@ -77,22 +80,27 @@ const NavBar: React.FC = () => {
         )}
         <Link
           to="/"
-          style={{ display: 'inline-flex' }}
+          style={{ display: 'inline-flex', zIndex: 10000 }}
```

Comment:
manually moving the logo to be on top of the absolutely-positioned div that wraps the search bar so the logo is clickable/not covered

Reply:
An alternative would be to wrap each section in flex containers and put the absolutely positioned div inside the middle container

```html
<flex-container>
  <flex-item>
    {/* Logo */}
  </flex-item>
  <flex-item>
    { renderSearch && renderSearch() }
  </flex-item>
  <flex-item>
    { renderMenuItems && renderMenuItems() }
  </flex-item>
</flex-container>
```

in this case, `renderSearch` should be responsible of wrapping the absolutely positioned div in a positioned one, that way the `NavBar` doesn't "need to know" that some of the render props are using position absolute

```tsx
const renderSearch = () => (
  <ContainerPositionRelative>
    <Search {...props} />
  </ContainerPositionRelative>
);
```

For `renderSecondaryElement`, we could just make it so it renders whatever is after the search, that way we don't need to special-case the donate button

```tsx
const renderMenuItems: () => (
  <>
    <DonateButton />
    <HamburgerMenu />
  </>
)
```

or 

```tsx
const renderMenuItems: () => (
    <HamburgerMenu />
)
```

Reply:
removed special casing of the donate button. re: the first part (wrapping each in flex item)--search needs to be in an absolutely positioned div that the navbar items are aware of, so that it sits in the middle of the navbar and not in the middle of its flex container (would be off-center)